src/main/java/uk/ac/cam/cl/dtg/segue/auth/SegueLocalAuthenticator.java

```diff
@@ -244,8 +244,13 @@ public void ensureValidPassword(final String password) throws InvalidPasswordExc
             throw new InvalidPasswordException("Invalid password. You cannot have an empty password.");
         }
 
-        if (password.length() < 6) {
-            throw new InvalidPasswordException("Password must be at least 6 characters in length.");
+        if (password.length() < 12) {
```

Comment:
Rather than having this as a magic number, it would be better to save this value in a constant, the same way it's done with `SHORT_KEY_LENGTH`, for instance. Something like `MINIMUM_PASSWORD_LENGTH`, or similar.